target/linux/ramips/dts/rt3352_zte_mf283plus.dts

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+#include "rt3352.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "zte,mf283plus", "ralink,rt3352-soc";
+	model = "ZTE MF283+";
+
+	aliases {
+		led-boot = &led_wwan_green;
+		led-failsafe = &led_wwan_red;
+		led-upgrade = &led_wwan_red;
+		label-mac-device = &wmac;
```

Comment:
This only works if you assign the address with mtd-mac-address. Use label_mac in 02_network instead.

Reply:
Or bind it to Ethernet device device instead - as stated in commit message, both addresses in 'factory' partition were identical, despite being in different places. I determined which one is which by patching this partition.

While we are at that, maybe we should artificially shift one of the addresses? OTOH, as there is no "wan" port per se on this device, other than `wwan0`, and additional "WAN" label on LAN1 port used in non-default configuration in vendor image, it shouldn't hurt to leave them identical, as both are typically on same interface.

What do you think?

Reply:
> Or bind it to Ethernet device device instead - as stated in commit message, both addresses in 'factory' partition were identical, despite being in different places. I determined which one is which by patching this partition.

If that's the case, then yes, label-mac-device is preferred where possible.

> While we are at that, maybe we should artificially shift one of the addresses?

It's policy to mirror the vendor assignment as far as possible and reasonable. Changes may always be done by the users later.

Reply:
Ok, so let's use Ethernet as label device then. I'll post update shortly, of course I need to rebuild and test, and sysupgrade takes a looooong time on this device at the moment.

Reply:
If you just change the label-mac-device, I could live without a test.

Reply:
I have a fast machine dedicated for building, so it's still a matter of minutes ;-)

Reply:
```
root@OpenWrt:~# source /lib/functions/system.sh 
root@OpenWrt:~# get_mac_label
8c:79:67:70:25:c2
root@OpenWrt:~#
```
It works, so resolving.